client/src/components/chess-game/ChessGame.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import ChessJs from 'chess.js';
+import { Prompt } from 'react-router-dom';
```

Comment:
Can't we achieve this with window.prompt?
https://developer.mozilla.org/en-US/docs/Web/API/Window/prompt

Reply:
Yes, we can. But I felt this was much cleaner.